app/src/main/kotlin/org/cru/godtools/ui/dashboard/DashboardLayout.kt

```diff
@@ -87,6 +94,33 @@ internal fun DashboardLayout(onEvent: (DashboardEvent) -> Unit, viewModel: Dashb
     val snackbarHostState = remember { SnackbarHostState() }
     AppUpdateSnackbar(snackbarHostState)
 
+    // region optInNotification
+    val showOverlay by viewModel.showOptInNotification.collectAsState()
+
+    val context = LocalContext.current
+    val activity = context as? Activity
```

Comment:
there is a `LocalActivity.current` you can use here instead of casting `context` to an Activity